.github/workflows/python.yml

```diff
@@ -119,9 +119,14 @@ jobs:
           fi
 
           echo "checking ${python_files[@]}"
-          # workaround to get mypy to figure out the types it should install
-          mypy --config-file mypy.ini . > /dev/null 2> /dev/null || true
-          mypy --install-types --non-interactive
+
+          if [ -f ".mypy-stub-pkgs" ]; then
+            pip install -r .mypy-stub-pkgs
+          else
+            # workaround to get mypy to figure out the types it should install
+            mypy --config-file mypy.ini . > /dev/null 2> /dev/null || true
+            mypy --install-types --non-interactive
```

Comment:
Out of curiosity, should this block not be executed if `.mypy-stub-pkgs` exists? As far as I tried locally, `mypy --install-types` doesn't automatically upgrade `types-*` modules if they are already installed.

Reply:
Good point. I'll make the change.

Reply:
Done in e15fdf1ce2038d07e4765db9667ead67f42cf1c1